common/components/WalletDecrypt/components/DeterministicWalletsModal.tsx

```diff
@@ -42,11 +42,11 @@ interface Props {
 
   onCancel(): void;
   onConfirmAddress(address: string, addressIndex: number): void;
-  onPathChange(path: string): void;
+  onPathChange(dPath: DPath): void;
 }
 
 interface State {
-  currentLabel: string;
+  currentDPath: DPath;
```

Comment:
This seems like an invalid type since if the user uses a custom dpath, it's not in our enumeration of dpaths.

Reply:
`DPath` isn't an enum, it's just an interface. So it's OK if it's not one of our hardcoded ones.